### DISCUSSION TITLE
Use control flow for commands

### DISCUSSION THREAD

**Discussion by theogf** [i3/i3#5028] [Q&A] [answered]
Related to my question here https://www.reddit.com/r/i3wm/comments/vixany/how_to_cycle_through_windows_with_the_workspace/
How should one go about using control flow for commands?
For example (related to my question here), how do I get a keybind to do something different in function of which workspace I am currently focused on?

EDIT: I see now this related to https://github.com/i3/i3/discussions/5018 (did not appear in my search), but any pointer on how to use IPC would be welcome

**Comment by Airblader** [comment, marked as answer]
You bind the key to run a script and that script can do whatever you need it to do. You can use the IPC provided by i3 to check something like the focused workspace.
